tools/generate_project.py

```diff
@@ -0,0 +1,582 @@
+#!/usr/bin/env python3
+"""Tool to create a SLCP project in a Simplicity Studio build directory."""
```

Comment:
This does not only generate but also build projects from what I can see? Wasn't quite clear from the name :sweat_smile: